crates/context_aware_config/migrations/2024-11-05-083200_add_priority_restructure/up.sql

```diff
@@ -0,0 +1,8 @@
+-- Your SQL goes here
+ALTER TABLE public.dimensions
```

Comment:
Can you backfill this based on priority? Can be computed from priority

Reply:
we can but , serial order is being not followed in all tenants and their values does not need be of 2's power